src/ui/screens/Tips/TipDetailScreen.tsx

```diff
@@ -69,6 +72,7 @@ function TipDetailScreen({route}: ScreenProps) {
   const id = route.params?.id;
   const {data: tip, loading, refetching, refetch} = useTip(id);
   const {colors} = useTheme();
+  const navigation = useNavigation<NavigationProp<AppStackParamList>>();
```

Comment:
NIT 💅 

If a component is a screen (Part of app's navigation) we already have the `navigation` object in the props. We should use that instead of the `useNavigation` hook.